src/survey/views.py

```diff
@@ -45,15 +66,15 @@ def get(self, request, id):
             self.serializer_class(survey[0]).data, status.HTTP_200_OK
         )
 
-    def post(self, request, id):
```

Comment:
Are we going to make use of these `match_id` parameters for `post` and `delete` in this PR? If not, make this change in the PR where you'll actually make use of them.

Reply:
`match_id` is irrelevant but it's in the URL structure to maintain consistency with the other endpoints (i.e. creating a survey, for which you do need `match_id`). Because it's in the URL structure, it has to be in the method header because otherwise Django will try to pass it in and Python will get mad
tl;dr: i removed the parameters i didn't need and it raised an error